ai-decision-services/ai-ad-dpp-engine/run-tests/run-tests.md

```diff
@@ -0,0 +1,280 @@
+Lab 8: A simple data processing, training and inferencing example
+===
+
+## Training
+
+To make sure you have everything setup well in training, you can upload the training data to the **training-data-bucket**. If you see a Data Flow run is triggered is successful, and under Anomaly Detection a model is trained successfully, then the training path is working.
```

Comment:
Add the expected sequence to check instead of all in one line

1. Look for a new Data Flow Run created under the application
2. Wait for the Run status to change from Accepted to In Progress and then Successful. Note: The transition may sometimes happen before you navigate and check
3. If the Run moves to Failed, check the logs as 
4. Once the Run succeeds, a new AD model should get trained successfully. Look for a file **model_info** under <> folder in the **results bucket**. If the training has failed, search for the model OCID from the search box or by navigating to the Anomaly Detection project, and check the reason for failure.